pkg/command/utils/htmltemplatebindata.go

```diff
@@ -8,6 +8,7 @@ package utils
 import (
 	"bytes"
 	"compress/gzip"
+	"crypto/sha256"
```

Comment:
The generated go file shouldn't be changed unless `https://github.com/knative-sandbox/kperf/blob/master/templates/single_chart.html` is updated.

Reply:
Thanks, fixed.